import/plugin.h

```diff
@@ -0,0 +1,61 @@
+/****************************************************************************
+**
+** BSD 3-Clause License.  See LICENSE for full text.
+** Copyright (c) 2021 Open Mobile Platform LLC.
+** All rights reserved.
+**
+****************************************************************************/
+
+#ifndef AMBER_WEB_AUTHORIZATION_PLUGIN_H
+#define AMBER_WEB_AUTHORIZATION_PLUGIN_H
+
+#include <QtCore/QCoreApplication>
+#include <QtCore/QTranslator>
+#include <QtCore/QLocale>
+
+#include <QtQml/QQmlExtensionPlugin>
+
+namespace Amber {
+
+namespace Web {
+
+namespace Authorization {
+
+class AmberWebAuthorizationPlugin : public QQmlExtensionPlugin
+{
+    Q_OBJECT
+    Q_PLUGIN_METADATA(IID "org.qt-project.Qt.QQmlExtensionInterface")
```

Comment:
Should it be named somehow differently?

Reply:
this one describes to the plugin loader that it's a QQmlExtensionPlugin, without requiring the loader to load the plugin itself (instead it can inspect the metadata).